wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -200,6 +206,24 @@ pub enum WalletCommand {
         additional_metadata_uri: Option<String>,
     },
 
+    /// Mint new tokens and increase the total supply
+    MintTokens {
+        token_id: String,
+        address: String,
+        amount: String,
+    },
+
+    /// Unmint existing tokens and reduce the total supply
+    UnmintTokens {
+        token_id: String,
+        amount: String,
+    },
+
+    /// Lock the total supply for the tokens
```

Comment:
it's rather "Lock the circulating supply for the token" because total supply cannot be changed anyway